pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -310,10 +313,11 @@ def containerRaidLevelsSupported(device_type):
     """
     if device_type in (DEVICE_TYPE_LVM, DEVICE_TYPE_LVM_THINP):
         supported = set(raid.RAIDLevels(["raid0", "raid1", "raid4", "raid5", "raid6", "raid10"]))
-        return get_supported_raid_levels(DEVICE_TYPE_MD).intersection(supported).union(set([None]))
+        return get_blivet_supported_raid_levels(DEVICE_TYPE_MD).intersection(supported).union({None})
```

Comment:
You are over the character limit here too.

Reply:
Fixed.